<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitInspector</class>
 <widget class="QWidget" name="FitInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="plot_tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="ParameterFitPlotter" name="fit_plots">
       <attribute name="title">
        <string>Parameter fits</string>
       </attribute>
      </widget>
      <widget class="DeviationBoxPlot" name="deviation_boxplot">
       <attribute name="title">
        <string>Param deviation summary</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QSplitter" name="sidebar">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QPushButton" name="copy">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="paste">
          <property name="text">
           <string>Paste</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="sepcols">
          <property name="text">
           <string>Colours</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QTableWidget" name="fits">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideLeft</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>26</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>10</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>25</number>
          </attribute>
          <column>
           <property name="text">
            <string>p</string>
           </property>
           <property name="toolTip">
            <string>Parameter colour</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>e</string>
           </property>
           <property name="toolTip">
            <string>Error colour</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>D</string>
           </property>
           <property name="toolTip">
            <string>Deck number</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ep</string>
           </property>
           <property name="toolTip">
            <string>Epochs</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>o</string>
           </property>
           <property name="toolTip">
            <string>Stimulation order</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>xov</string>
           </property>
           <property name="toolTip">
            <string>Crossover</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>a</string>
           </property>
           <property name="toolTip">
            <string>Annealing</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>t</string>
           </property>
           <property name="toolTip">
            <string>Target type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>m</string>
           </property>
           <property name="toolTip">
            <string>Method</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>c</string>
           </property>
           <property name="toolTip">
            <string>Clustering</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>s</string>
           </property>
           <property name="toolTip">
            <string>Mutation selectivity (0: nonspecific, 1: graded, 2: target only)</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QWidget" name="settingsButtons" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="rundata">
             <property name="text">
              <string>RunData</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="daqdata">
             <property name="text">
              <string>DAQData</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="fittersettings">
             <property name="text">
              <string>FitterSettings</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" colspan="4">
         <widget class="QTableWidget" name="groups">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>15</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>10</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>25</number>
          </attribute>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="addGroup">
          <property name="text">
           <string>Add group</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="delGroup">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="saveGroups">
          <property name="text">
           <string>Save...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="loadGroups">
          <property name="text">
           <string>Load...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParameterFitPlotter</class>
   <extends>QWidget</extends>
   <header>parameterfitplotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DeviationBoxPlot</class>
   <extends>QWidget</extends>
   <header>deviationboxplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
